builder/builder-next/builder.go

```diff
@@ -32,7 +32,10 @@ import (
 	grpcmetadata "google.golang.org/grpc/metadata"
 )
 
-var errMultipleFilterValues = errors.New("filters expect only one value")
+var (
+	errMultipleFilterValues = errors.New("filters expect only one value")
```

Comment:
Both op these should result in a 4xx status to be returned; can you make these errors implement [`ErrInvalidParameter`](https://github.com/moby/moby/blob/d48392a35b157612d28c48eb8e4fad1272fa1442/errdefs/defs.go#L8-L11)?